packages/modals/src/elements/Modal.tsx

```diff
@@ -71,6 +71,14 @@ export interface IModalProps {
    * Enable large modal styling
    */
   isLarge?: boolean;
+  /**
+   * Determines whether to put keyboard focus on the modal on mount
+   */
+  hasFocusOnMount?: boolean;
```

Comment:
With `focus/restore` being actions could we leave theme as `focusOnMount`/`restoreFocus`? Feels clunky to with the `has/is` prefix.

Reply:
These variables use `has/is` prefix to follow the heuristic that all non-native boolean attributes should use those prefixes. I could rename these if "action variables" don't get `has/is` prefixes. Would these be the first booleans in Garden that don't have those prefixes?

Reply:
Looks like I need to update my heuristics when it comes to boolean naming as this is already an established practice: `focusInset`.